modules/core/package.json

```diff
@@ -24,9 +24,11 @@
   "browser": {
     "fs": false,
     "stream": false,
-    "./src/iterators/make-stream/make-node-stream.ts": false,
-    "./src/iterators/make-stream/make-node-stream.js": false,
-    "./dist/iterators/make-stream/make-node-stream.js": false
+    "./src/iterators/make-stream/make-node-stream.ts": "./src/iterators/make-stream/make-dom-stream.ts",
+    "./src/iterators/make-stream/make-node-stream.js": "./src/iterators/make-stream/make-dom-stream.js",
+    "./dist/iterators/make-stream/make-node-stream.js": "./dist/iterators/make-stream/make-dom-stream.js",
```

Comment:
This is not a valid path - who's importing it?

Reply:
Master is targeting 4.0.0-alpha which drops the dual builds

Reply:
Then it shouldn't have the dist/es5 and dist/esm paths. You need to make separate changes to master and 3.1-release.

Reply:
Since this is still being experimented with I though it wouldn't hurt to have all of them. But maybe it does confuse the bundler.

Reply:
@ibgreen @Pessimistress I don't know if it forsure confuses the bundler, but I can say that personally I find it very confusing.